pkg/actions/project.go

```diff
@@ -284,3 +284,155 @@ func ProjectRestart(c *cli.Context) {
 	fmt.Println(string(response))
 	os.Exit(0)
 }
+
+// ProjectLinkList : lists all the links for a project
+func ProjectLinkList(c *cli.Context) {
+	projectID := strings.TrimSpace(strings.ToLower(c.String("id")))
+
+	conID, getConnectionIDErr := project.GetConnectionID(projectID)
```

Comment:
not a you thing, but noticed testing the commands that when I give a non existent project I get a file read error (in /.codewind/connections). We could perhaps catch this and return something clearer.